docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -73,7 +73,7 @@ following limitations currently exist for SOA Suite domains:
   been added yet.
 
 {{% notice note %}}
-For early access customers, with bundle patch access, it is recommended to build and use the Oracle SOA Suite Docker image with the latest bundle patches for Oracle SOA and Oracle Service Bus (OSB). The Oracle SOA Suite Docker image in `container-registry.oracle.com` does not have the bundle patches installed. However the customers who do not have access to the bundle patches can obtain the Oracle SOA Suite Docker image from `container-registry.oracle.com` as described in below section.
+For early access customers, with bundle patch access, we recommend that you build and use the Oracle SOA Suite Docker image with the latest bundle patch for Oracle SOA. The Oracle SOA Suite Docker image in `container-registry.oracle.com` does not have the bundle patch installed. However, if you do not have access to the bundle patches, you can obtain the Oracle SOA Suite Docker image without bundle patches from `container-registry.oracle.com`, as described below.
```

Comment:
if you do not have access to the bundle patches -> if you do not have access to the bundle patch
without bundle patches from -> without the bundle patch from